cmd/sup/main.go

```diff
@@ -99,6 +100,12 @@ func parseArgs(conf *sup.Supfile) (*sup.Network, []*sup.Command, error) {
 	// Add default env variable with current network
 	network.Env["SUP_NETWORK"] = args[0]
 
+	// Add default nonce
+	network.Env["SUP_NONCE"] = time.Now().Format("2006-01-02T15:04:05")
```

Comment:
I like this PR too. And I'd like to see even more ENV VARs like this:

`SUP_TIME` (instead of "SUP_NONCE") - UTC timestamp in RFC3339 format
`SUP_UNIXTIME` - Unix time (number of seconds since 1970)
`SUP_USER` - user who invoked the sup command